pkg/build/webhook/github/github.go

```diff
@@ -53,8 +53,8 @@ func (p *WebHook) Extract(buildCfg *api.BuildConfig, secret, path string, req *h
 		return revision, envvars, dockerStrategyOptions, proceed, err
 	}
 	method := getEvent(req.Header)
-	if method != "ping" && method != "push" && method != "Push Hook" {
```

Comment:
where did "Push Hook" come from as a method value and what makes us confident in removing it?  Was it part of the gitlab support this hook apparently had?